### ISSUE TITLE
We need an Icon

### ISSUE THREAD

**Issue by hahn-kev** [sillsdev/languageforge-lexbox#60]

**Comment by rmunn** [comment]
Off-the-cuff idea, and I don't know that I like it much because basic geometric shapes with letters on them already feature in so many of our products... but maybe a cube with the letters L, E, and X on the three visible sides of the cube?

**Comment by hahn-kev** [comment]
I like Garden and Library, though one thing Chris mentioned when I talked to him is that there's a lot of confusion between Language Forge and Language Depot, just because the names are so similar, so something that gets away from Language something would be better at giving it a distinct title. I kinda like Dictionary Depot because it keeps Depot in the name for users to track that it's the same thing that it used to be.

**Comment by hahn-kev** [comment]
Some other names I came up with

- codex depot
- dictionary depot

**Comment by jasonleenaylor** [comment]
I kind of like Language Locker (trademark available, .com squatted, .org free, .app free)

Pros:

- Makes you think of securely keeping your data
- Not dictionary specific

Cons:

- Doesn't make you think of sharing the data

**Comment by hahn-kev** [comment]
Please submit any new suggestions by the 9th, and submit a ranked list of 3 names from this issue by the 11th. Next Friday we will discuss them and try to find our favorite name as a group using that ranked list of names.

**Comment by alex-larkin** [comment]
How about LexVerse?

**Comment by alex-larkin** [comment]
I also like LexDrive

**Comment by alex-larkin** [comment]
Some ideas from ChatGPT, or inspired from ChatGPT ideas:

FolkFusion
FolkLoom
LexLoom
LexWeave
LexLore
LexiLink
LexiWeave
LingoLoom
LingoWeave
LingoNet
TalkTrove

I tried to get it to give me some single-concept ideas like Twitter or Bloom, but it just regurgitated more of the same 😂

**Comment by alex-larkin** [comment]
LexNest

**Comment by jasonleenaylor** [comment]
Looking at this issue again to see what the options are I have two thoughts:
1. How much of the confusion between Language Depot and Language Forge came because of the Language part, and how much because of inadequate branding on Language Depot.
2. Names that focus on lexical data kind of leaves [OSE](https://onestory.org) in the cold

**Comment by hahn-kev** [comment]
Yeah we talked about OSE, it's kinda the outlier as far as data storage goes so it does make naming difficult if everything needs to fit under the name.

As far as the confusion, I think it's just the similarity in the name as even people very well acquainted with the products slip up and use the wrong name.

**Comment by hahn-kev** [comment]
Alright, we might be getting out of hand with all these suggestions. Let's keep it to 5 per person.

**Comment by myieye** [comment]
> Here's a few more:
> 
> WordWeave LingoLink LinguaLoom
> 
> Here are my sessions if anyone is curious
> 
> https://chat.openai.com/share/900e523f-0a1d-4333-a308-2422fb017fef
> 
> https://chat.openai.com/share/88610493-ec5e-44c3-ba1e-1c53618b4f18
> 
> I need to do some more vetting to check domain availability.
> 
> Also, I imagine we need to check if these terms are trademarked

@alex-larkin You're a lot more polite than me when you interact with ChatGPT 😆

**Comment by josephmyers** [comment]
My two-cents, for more single-concept ideas:
- glossia.org (Greek for "language" or "tongue")
- vernacular.org (I'm iffy on this, but it is available)
- arborofgod.org (for garden theme!)

I'm a big fan of glossia. It's flashy, concise, and mysterious in a good way. Great brand name. It's almost too big for LD, haha, but I would totally swipe it.

**Comment by myieye** [comment]
@josephmyers glossia is really catchy.
Can you give us a link regarding the Greek meaning?
It seems to me that "Glossa" is a Greek work, but "glossia" is an English suffix based on "Glossa".

**Comment by alex-larkin** [comment]
I really like Glossia too.

Regarding ChatGPT politeness, I've tried being more curt but the cognitive dissonance it causes me makes me prefer to put in the occasional "please", even though I know it's not needed 😂

**Comment by alex-larkin** [comment]
"Vernacular" and it's cognates in other languages can have a negative connotation. Also, at least in minority language contexts, it's a less common term that not everyone knows. So those factors make me hesitant to use that term.

**Comment by alex-larkin** [comment]
After paring down my suggestions, I checked the last seven or so for trademarks and domain availability. Then I did a Google search on the remaining terms to see what was out there. After that filtering, I am left with just one suggestion: 

**LingoWeave**

The .com, .org, and .net domains are free. I couldn't find any US trademarks (after a quick search on their database) and I didn't get any relevant Google search results for either "Lingoweave" or "Lingo weave". 

I affirm that I really like some of the other names ya'll have submitted, and will probably be ranking them highly too!

**Comment by josephmyers** [comment]
> 

I just googled "glossia in greek," and Google picked it up, but it does appear that the actual Greek is "glossa," like you said. More accurate to say it's the suffix for "language" and "tongue," coming from the Greek root. (It's still a great pick, though :D)

**Comment by josephmyers** [comment]
> "Vernacular" and it's cognates in other languages can have a negative connotation. Also, at least in minority language contexts, it's a less common term that not everyone knows. So those factors make me hesitant to use that term.

Yeah, I was iffy on it, too. Maybe that's why it's available...

**Comment by hahn-kev** [comment]
Final list of names:

- ~~Glossia~~ there's already a trade mark for this name
- LDrive
- Open Ark
- [The Open] Language Garden
- Language Locker

**Comment by hahn-kev** [comment]
We've got a brand! LexBox is it. Now we need to update some stuff for the new name
